Makefile.dep

```diff
@@ -967,6 +967,11 @@ ifneq (,$(filter usbus_cdc_acm,$(USEMODULE)))
   USEMODULE += usbus
 endif
 
+ifneq (,$(filter usbus_hid,$(USEMODULE)))
+  USEMODULE += isrpipe_read_timeout
+  USEMODULE += usbus
+endif
+
```

Comment:
You shoud move this to sys/Makefile.dep after rebasing this PR on top of master.

Reply:
done